pry/pry.go

```
@@ -52,12 +52,18 @@ func apply(
 
 	displayFilePosition(out, filePathRaw, filePath, lineNum)
 
-	history := loadHistory()
+	history, err := NewHistory()
+	if err != nil {
+		fmt.Errorf("Failed to initiliaze history %+v", err)
+	}
+	if err := history.Load(); err != nil {
+		fmt.Errorf("Failed to load the history %+v", err)
```

you can also just return an error here with errors.Wrapf